applications/callflow/src/cf_exe.erl

```diff
@@ -668,14 +668,20 @@ terminate({'shutdown', 'control_usurped'}, _) ->
     lager:info("the call has been usurped by an external process");
 terminate(_Reason, #state{call=Call
                          ,cf_module_pid='undefined'
+                         ,hangup_info=HangupInfo
```

Comment:
handle `undefined` to determine if there's a need to send hangup

Reply:
This is being handled in `maybe_hangup_call/1` now.